docs/JLBP-0016.md

```diff
@@ -5,8 +5,7 @@ permalink: /JLBP-16
 ---
 # Ensure upper version alignment of dependencies for consumers
 
-Upper version alignment means that the version of each dependency 
-added to the classpath is the highest version in the dependency tree.
+The version of each dependency added to the classpath should the highest version in the dependency tree.
 Upper version alignment ensures that upgrading packages to compatible higher versions doesn't introduce new linkage conflicts.
```

Comment:
```suggestion
This upper version alignment ensures that upgrading packages to compatible higher versions doesn't introduce new linkage conflicts.
```

How about this?

Reply:
I don't know. Reads a little awkward to me. maybe it impies there's some other upper version alignment in play?